### ISSUE TITLE
Scheduler as config

### ISSUE THREAD

**Issue by lvoegtlin** [DIVA-DIA/DIVA-DAF#38]
**Is your feature request related to a problem? Please describe.**
Provide the LR scheduler to the task by config. As the loading part of the scheduler is already done this should work like the loss #14 

**Describe the solution you'd like**
Define the LRScheduler object and initialize it in the task
A good scheduler would be [CosineAnnealingLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingLR.html#torch.optim.lr_scheduler.CosineAnnealingLR)

### Checklist

- [ ] Create config for two schedulers
- [ ] Get class pointer during startup and hand it over to the task
- [ ] Task create an object based on the class pointer and the provided arguments for the scheduler
